code/modules/mob/living/simple_animal/abnormality/he/woodsman.dm

```diff
@@ -76,7 +78,7 @@
 		return ..()
 	var/list/highest_priority = list()
 	var/list/lower_priority = list()
-	for(var/mob/living/L in Targets)
+	for(var/mob/living/L in view(8, src))
```

Comment:
..?

Reply:
I was confused as well looking back on this, and I remember WHY I did this now. HE WASN'T TARGETTING DEAD BODIES. AT ALL. This just overwrites that so he does.

Reply:
That's because he isn't set to target dead people. Change stat_attack if you want it to, but as far as I remember, it isn't meant to.

Reply:
Gotcha